scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.csv

```diff
@@ -1,740 +1,758 @@
+246,mokokchung,13,nagaland,13.03,262,mokokchung,http://www.wikidata.org/entity/Q2175311,mokokchung,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,262,Q2175311,Mokokchung,Mokokchung,wikidataId/Q1599,wikidataId/Q2175311
+247,mon,13,nagaland,13.01,261,mon,http://www.wikidata.org/entity/Q2339648,mon,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,261,Q2339648,Mon,Mon,wikidataId/Q1599,wikidataId/Q2339648
+248,phek,13,nagaland,13.08,266,phek,http://www.wikidata.org/entity/Q590882,phek,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,266,Q590882,Phek,Phek,wikidataId/Q1599,wikidataId/Q590882
+249,tuensang,13,nagaland,13.02,267,tuensang,http://www.wikidata.org/entity/Q2571393,tuensang,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,267,Q2571393,Tuensang,Tuensang,wikidataId/Q1599,wikidataId/Q2571393
+250,wokha,13,nagaland,13.05,264,wokha,http://www.wikidata.org/entity/Q681821,wokha,district of Nagaland,http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,264,Q681821,Wokha,Wokha,wikidataId/Q1599,wikidataId/Q681821
+251,zunheboto,13,nagaland,13.04,263,zunheboto,http://www.wikidata.org/entity/Q2091461,zunheboto,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,263,Q2091461,Zunheboto,Zunheboto,wikidataId/Q1599,wikidataId/Q2091461
+613,peren,13,nagaland,13.07,271,peren,http://www.wikidata.org/entity/Q516294,peren,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,271,Q516294,Peren,Peren,wikidataId/Q1599,wikidataId/Q516294
+614,kiphire,13,nagaland,13.02,269,kiphire,http://www.wikidata.org/entity/Q2597908,kiphire,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,269,Q2597908,Kiphire,Kiphire,wikidataId/Q1599,wikidataId/Q2597908
+615,longleng,13,nagaland,13.02,268,longleng,http://www.wikidata.org/entity/Q1426783,longleng,"district of Nagaland, India",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,268,Q1426783,Longleng,Longleng,wikidataId/Q1599,wikidataId/Q1426783
+736,noklak,13,nagaland,,,noklak,http://www.wikidata.org/entity/Q48731903,noklak,"district in India, Nagaland",http://www.wikidata.org/entity/Q1599,nagaland,state in northeastern India,,Q48731903,Noklak,Noklak,wikidataId/Q1599,wikidataId/Q48731903
+757,tseminyu,13,nagaland,,,tseminyu,http://www.wikidata.org/entity/Q110223836,tseminyu,"district of Nagaland, India",http://www.wikidata.org/wntity/Q1599,nagaland,Indian state,,Q110223836,Tseminyu,Tseminyu,wikidataId/http://www.wikidata.org/wntity/Q1599,wikidataId/Q110223836
```

Comment:
pls remove url from the StatDCID column:
`wikidataId/http://www.wikidata.org/wntity/Q1599` -> `wikidataId/Q1599`

Reply:
Fixed.